### PR TITLE
hw-mgmt: patches: Update patches for kernel v6.1

### PR DESCRIPTION
**Pull request by vadimp-nvidia**
Reviewed-by: Oleksandr Shamray <oleksandrs@nvidia.com>

### CODE REVIEW

**Comment by vadimp-nvidia** [comment]
RERUN

**Comment by vadimp-nvidia** [comment]
RERUN

**Comment by ciju-nvidia** [comment]
RERUN

File: recipes-kernel/linux/linux-6.1/0039-mlxsw-reg-Limit-MTBR-register-payload-to-a-single-da.patch [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+diff --git a/drivers/net/ethernet/mellanox/mlxsw/core_env.c b/drivers/net/ethernet/mellanox/mlxsw/core_env.c
+index 0107cbc32fc7..550475a108ef 100644
+--- a/drivers/net/ethernet/mellanox/mlxsw/core_env.c
++++ b/drivers/net/ethernet/mellanox/mlxsw/core_env.c
+@@ -777,7 +777,7 @@ static int mlxsw_env_module_has_temp_sensor(struct mlxsw_core *mlxsw_core,
+ 	int err;
+ 
+ 	mlxsw_reg_mtbr_pack(mtbr_pl, slot_index,
+-			    MLXSW_REG_MTBR_BASE_MODULE_INDEX + module, 1);
```
Comment (mshych) [inline recipes-kernel/linux/linux-6.1/0039-mlxsw-reg-Limit-MTBR-register-payload-to-a-single-da.patch:32]: Is it an ifc change?

File: recipes-kernel/linux/linux-6.1/0013-platform-mellanox-mlx-platform-Fix-signals-polarity-.patch [hunk 2]
```diff
@@ -10,24 +10,26 @@ mask for L1 switch.
...
 --- a/drivers/platform/x86/mlx-platform.c
 +++ b/drivers/platform/x86/mlx-platform.c
-@@ -239,7 +239,7 @@
+@@ -237,7 +237,7 @@
+ #define MLXPLAT_CPLD_GWP_MASK		GENMASK(0, 0)
  #define MLXPLAT_CPLD_EROT_MASK		GENMASK(1, 0)
- #define MLXPLAT_CPLD_FU_CAP_MASK	GENMASK(1, 0)
  #define MLXPLAT_CPLD_PWR_BUTTON_MASK	BIT(0)
 -#define MLXPLAT_CPLD_LATCH_RST_MASK	BIT(5)
 +#define MLXPLAT_CPLD_LATCH_RST_MASK	BIT(6)
```
Comment (mshych) [inline recipes-kernel/linux/linux-6.1/0013-platform-mellanox-mlx-platform-Fix-signals-polarity-.patch:28]: Why is it just in 6.1?

**Review by sholeksandr — APPROVED** [review]

**Pull request merged**